components/Footer.js

```diff
@@ -0,0 +1,15 @@
+import Link from 'next/link'
+import styles from '../styles/Home.module.css'
+
+const Footer = () => (
```

Comment:
The guide I found for pages/components uses the 'fat arrow' function definition style, so it will look [different](https://www.sitepoint.com/arrow-functions-javascript/) than the other js files.